src/App.jsx

```diff
@@ -14,33 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const generateKey = () => {
+  return `_${Math.random().toString(36).substr(2, 9)}`;
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
```

Comment:
this state is also redundant, create the var
```suggestion
  const visibleGoods = getVisibleGoods(goodsFromServer, sortValue, isReversed);
```
create the getVisibleGoods function that will handle all logic with sorting and reversing the array